_includes/callback-reference.md

```diff
@@ -74,18 +82,18 @@ Swedbank Pay, and the two `GET` requests that you make to get the updated
 status.
 
 ```mermaid
-sequenceDiagram
-  Participant Merchant
-  Participant SwedbankPay as Swedbank Pay
+    sequenceDiagram
```

Comment:
```suggestion
sequenceDiagram
```
The first part isn't indented. The sequence diagram is a sub-element of this initial block.